### ISSUE TITLE
Bug: 

### ISSUE THREAD

**Issue by TL37** [nathom/streamrip#79]
## Bug Report

**Current Behavior**
Following upgrade from v0.5.1 to 0.5.3 using pip3, rip triggers an error and will not launch. Worked normally prior to running pip3 streamrip --upgrade. See traceback info below when rip is entered with any parameters. I uninstalled and reinstalled using pip3 with the same result.

**Command used:**

```bash
rip -u {url}, rip --config, etc.
```

```yaml

MacBook-Pro:streamrip xxxxx$ cat config.yaml 
qobuz:
  # 1: 320kbps MP3, 2: 16/44.1, 3: 24/<=96, 4: 24/>=96
  quality: 4
  # This will download booklet pdfs that are included with some albums
  download_booklets: true
  email: xxxxx
  # This is an md5 hash of the plaintext password
  password: xxxxx
  # Do not change
  app_id: '950096963'
  # Do not change
  secrets:
  - 10b251c286cfbf64d6b7105f253d9a2e
  - 979549437fcc4a3faad4867b5cd25dcb
tidal:
  # 0: 256kbps AAC, 1: 320kbps AAC, 2: 16/44.1 "HiFi" FLAC, 3: 24/44.1 "MQA" FLAC
  quality: 3
  # This will download videos included in Video Albums.
  download_videos: true
  # Do not change any of the fields below
  user_id:
  country_code:
  access_token:
  refresh_token:
  # Tokens last 1 week after refresh. This is the Unix timestamp of the expiration time.
  token_expiry: 0
# Deezer doesn't require login
deezer:
  # 0, 1, or 2
  quality: 2
soundcloud:
  # Only 0 is available
  quality: 0
youtube:
  # Only 0 is available for now
  quality: 0
  # Download the video along with the audio
  download_videos: false
  # The path to download the videos to
  video_downloads_folder: /Users/xxxxx/StreamripDownloads
# This stores a list of item IDs so that repeats are not downloaded.
database:
  enabled: true
  path: /Users/xxxxx/Library/Application Support/streamrip/downloads.db
conversion:
  enabled: false
  codec:
  sampling_rate:
  bit_depth:
# Filter a Qobuz artist's discography. Set to 'true' to turn on a filter.
filters:
  # Remove Collectors Editions, live recordings, etc.
  extras: false
  # Picks the highest quality out of albums with identical titles.
  repeats: false
  # Remove EPs and Singles
  non_albums: false
  # Remove albums whose artist is not the one requested
  features: false
  non_studio_albums: false
  # Only download remastered albums
  non_remaster: false
downloads:
  # Folder where tracks are downloaded to
  folder: /Users/xxxxx/StreamripDownloads
  # Put Qobuz albums in a 'Qobuz' folder, Tidal albums in 'Tidal' etc.
  source_subdirectories: false
artwork:
  # Write the image to the audio file
  embed: true
  # The size of the artwork to embed. Options: thumbnail, small, large, original. 'original' images can be up to 30MB, and may fail embedding. Using 'large' is recommended.
  size: large
  # Save the cover image at the highest quality as a seperate jpg file
  keep_hires_cover: true
# Only applicable for playlist downloads.
metadata:
  # Sets the value of the 'ALBUM' field in the metadata to the playlist's name. This is useful if your music library software organizes tracks based on album name.
  set_playlist_to_album: false
  # Replaces the original track's tracknumber with it's position in the playlist
  new_playlist_tracknumbers: true
# Changes the folder and file names generated by streamrip.
path_format:
  # Available keys: "albumartist", "title", "year", "bit_depth", "sampling_rate", and "container"
  folder: '{albumartist} - {title} ({year}) [{container}] [{bit_depth}B-{sampling_rate}kHz]'
  # Available keys: "tracknumber", "artist", "albumartist", "composer", and "title"
  track: '{tracknumber}. {artist} - {title}'
check_for_updates: true
# Last.fm playlists are downloaded by searching for the titles of the tracks
lastfm:
  # The source on which to search for the tracks.
  source: qobuz
# Download (and convert) tracks all at once, instead of sequentially. If you are converting the tracks, and/or have fast internet, this will substantially improve processing speed.
concurrent_downloads: false
```
**Environment**

- `streamrip` version(s): v0.5.3
- Python version: 3.9
- OS: MacOS 11.3.1
- M1 MacBook Pro

**Possible Solution**
n/a

**Additional context/Screenshots**

[Traceback.txt](https://github.com/nathom/streamrip/files/6481729/Traceback.txt)

**Comment by nathom** [comment]
Try installing `gmp`, and then reinstalling `pycryptodome`. 

Using Homebrew:

```
brew install gmp && pip3 uninstall pycryptodome -y && pip3 install pycryptodome
```

**Comment by TL37** [comment]
Done; the error is the same.

**Comment by nathom** [comment]
I'm not sure what the issue is, but it's definitely not with `streamrip`. I'd advise opening an issue on the [pycryptodome page](https://github.com/Legrandin/pycryptodome/issues) and finding help there. In the meanwhile, I'll make `pycryptodome` an optional dependency in the next version so that you can use the tool. Some Tidal MQA downloads won't work, though.

**Comment by nathom** [comment]
Fixed in `v0.5.4`.
